content/intro/pricing/rates.md

```diff
@@ -0,0 +1,89 @@
+---
+menu:
+  main:
+    parent: pricing
+title: Rates
+weight: 10
+---
+
+## cloud.gov is charged with 3 different components:
+
+### 1. cloud.gov Access Package
+
+The access package is the main fee charged by cloud.gov and it includes the following: 
+
+- API access, CLI access, Web UI access
+- Documentation/tutorial material (including continuing development)
+- Web-based support during business hours (platform performing as expected and available 24/7, problems addressed as-possible).
+- Allocation of underlying IaaS (AWS) instances 
+- OS Security updates and Regular hardening of OS image
+- Regular updates to platform security
+- Network security
+- ATO-ready documentation of system components (compliance-masonry form)
+- Routine scanning for infrastructure-level vulnerabilities/misconfiguration
+- Routine security updates for supported [buildpacks](#)
+- Self-service management of spaces and users within an organization
+- Platform maintenance and managed service expansion overhead
+
+<table>
+  <tr>
+    <th>Sandbox</th>
+    <th>Skunkworks</th>
+    <th>FISMA Low</th>
+    <th>FISMA Mod</th>
+  </tr>
+  <tr>
+    <td>
+      For .gov and .mil accounts to kick the tires. Capped usage, all managed services available, no paperwork required.
+    </td>
+    <td>
+      All the gov-facing services you want on apps.cloud.gov, low separation between teams. Usually one per agency/department. Sandbox caps are removed.
+    </td>
+    <td>
+      <em>3</em> public-facing systems, including all the environments needed to support each and DNS support.
+    </td>
+    <td>
+      <em>1</em> public-facing system, including all the environments needed to support each and DNS support with the additional support required for FISMA Moderate payloads.
+    </td>
+  </tr>
+  <tr>
+    <th>Free</th>
+    <th>$15K / year</th>
+    <th>$60K / year</th>
+    <th>$90K / year</th>
+  </tr>
+</table>
```

Comment:
One of our biggest challenges is making sure people don't compare this to other public PaaSes as apples to apples since it isn't. The PaaS is just a hack to get us through the compliance part.

So I think keeping at least some of this is critical to the purpose of this page...